### ISSUE TITLE
Question about the schedule

### ISSUE THREAD

**Issue by zhanggang001** [wxzs5/direct-pretraining#3]
Hi, Yang Li, thanks for your wonderful work. I have one question about the schedule.
I see that you use RepeatDataset in the dataset and set times to 4, does this mean that the P1x actually trains the model by 4x (48epoch) Thank you in advance!

**Comment by wxzs5** [comment]
Yes, we keep it trained with same iterations (~90k) as baseline (batch size=2) for every epoch.

**Comment by zhanggang001** [comment]
Thank you. Another question about Table 3 in the paper.  Why is the total cost of Direct(P4x) less than Direct(P2x)? Is any error here? Actually, I run the code of Mask RCNN on 8 2080Ti GPUs and find that P1x needs roughly the time for fine-tuning 2x from ImageNet pre-trained model.

**Comment by wxzs5** [comment]
Yes, Fig. 1 is correct for that. We will release a new version paper to correct these errors.

**Comment by zhanggang001** [comment]
Again, thanks for your response!
